### PR TITLE
Add GQLSTATUS code to Neo. code

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-2008.surge.sh

Updated pages:
- [Concurrent data access](https://neo4j-docs-operations-2008.surge.sh/operations-manual/5/database-internals/concurrent-data-access)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-2008.surge.sh

Updated pages:
- [Database privileges](https://neo4j-docs-operations-2008.surge.sh/operations-manual/2025.01/authentication-authorization/database-administration)
- [Managing database aliases in composite databases](https://neo4j-docs-operations-2008.surge.sh/operations-manual/2025.01/database-administration/aliases/manage-aliases-composite-databases)
- [Concurrent data access](https://neo4j-docs-operations-2008.surge.sh/operations-manual/2025.01/database-internals/concurrent-data-access)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-2008.surge.sh

Updated pages:
- [Database privileges](https://neo4j-docs-operations-2008.surge.sh/operations-manual/2025.01/authentication-authorization/database-administration)
- [Managing database aliases in composite databases](https://neo4j-docs-operations-2008.surge.sh/operations-manual/2025.01/database-administration/aliases/manage-aliases-composite-databases)
- [Concurrent data access](https://neo4j-docs-operations-2008.surge.sh/operations-manual/2025.01/database-internals/concurrent-data-access)

File: modules/ROOT/pages/database-administration/aliases/manage-aliases-composite-databases.adoc [hunk 1]
```diff
@@ -354,6 +356,8 @@ DROP ALIAS $aliasname FOR DATABASE
 Failed to delete the specified database alias 'foo.bar': Database alias does not exist.
 ----
 
+//From 5.26 onwards, the error message also contains the GQLSTATUS code `50N00` and the status description `error: general processing exception - internal error. Internal exception raised { $msgTitle }: Failed to create the specified database alias 'foo.bar'. Database alias does not exist.`
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/database-administration/aliases/manage-aliases-composite-databases.adoc:359]: Should the GQLSTATUS `50N00` be hidden or not?

**Review by NataliaIvakina — APPROVED** [review]
Looks good! Only one question about the GQL code `50N00`

**Comment by renetapopova** [comment]
> Looks good! The GQL code `50N00` is hidden on purpose, I suppose

Yes, I don't know what the returned message is exactly.

**Pull request merged**